### PR TITLE
Fixed CodeInspector issues.

### PR DESCRIPTION
**Pull request by utoni**
* Removed compiler search list for AC_PROG_CC, AM_PROG_CC_C_O, AC_PROG_CXX, AC_PROG_CC_STDC
   The list rendered the AX_CHECK_COMPILE_FLAG([-fsanitize=fuzzer]) useless as it did use
   clang for AX_CHECK_COMPILE_FLAG but gcc during the compile process. Seems broken somehow.

Signed-off-by: Toni Uhlig <matzeton@googlemail.com>

### CODE REVIEW

File: configure.seed [hunk 1]
```diff
@@ -13,10 +13,10 @@ AS_IF([test "${with_sanitizer+set}" = set],[
...
 LT_INIT
 
-AC_PROG_CC(clang gcc)
-AM_PROG_CC_C_O(clang gcc)
-AC_PROG_CXX(clang++ g++)
-AC_PROG_CC_STDC(clang gcc)
+AC_PROG_CC
+AM_PROG_CC_C_O
+AC_PROG_CXX
+AC_PROG_CC_STDC
```
Comment (lucaderi) [inline configure.seed]: Doing this change, on my Mac clang won't be used. I don't think it's a good idea to revert the change. What do you think?

Comment (utoni) [reply configure.seed]: Strange. Can you post/upload `config.log` w/ and w/o those changes?
Would help me a lot.

**Comment by lucaderi** [comment]
[config.zip](https://github.com/ntop/nDPI/files/4866934/config.zip)

File: configure.seed (hunk 1 above)
Comment (lucaderi) [reply configure.seed]: Attached

Comment (utoni) [reply configure.seed]: Thanks, will investigate.

Comment (utoni) [reply configure.seed]: keeping my fingers crossed that it will work now

**Comment by lucaderi** [comment]
Thanks
